src/main/java/skibidi/Storage.java

```diff
@@ -0,0 +1,73 @@
+package skibidi;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import skibidi.task.AbstractTask;
+
+/**
+ * Handles writing tasks to disk and loading tasks from file.
+ */
+public final class Storage {
+    private String dataPath;
+
+    public Storage(String dataPath) {
+        this.dataPath = dataPath;
+    }
+
+    /**
+     * Save given task list to storage data path. Overrides any existing data on the file.
+     */
+    public void saveTasksToDisk(TaskList taskList) {
```

Comment:
i like this descriptive method name, here and throughout your code

Reply:
Thanks!